### ISSUE TITLE
SummaryBoxItem violates semantics and accessibility principles

### ISSUE THREAD

**Issue by sarahsehr** [appfolio/react-gears#357]
https://github.com/appfolio/react-gears/blob/master/src/components/SummaryBoxItem.js#L15

 Headings should be used to communicate structure because it affects the TOC for the page.  

We should not use HTMl tags for their style, but for what they semantically represent. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements
